### PR TITLE
Glowsilk Bow... Finally

### PR DESCRIPTION
**Pull request by Dariensg**
-Added Glowsilk
-Added Glowsilk Bow
-Added Glowsilk Cocoon
-Glowsilk Bow arrows give entities Luminous
-Created Glowsilk Cocoon Ore Feature
-Glowsilk Cocoons now generate in GSC, Deltas, and Crimson Forests between y-10 and y-30
-Arrows with Luminous or Infection now use the correct particles
-Added Infection potion brewing
-Added Strong and Long versions of Infection potions
-Fixed base custom potions being able to brew directly to lingering

### CODE REVIEW

File: src/main/java/com/nekomaster1000/infernalexp/items/GlowsilkBowItem.java [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+						if (abstractarrowentity instanceof ArrowEntity) {
+							((ArrowEntity) abstractarrowentity).addEffect(new EffectInstance(IEEffects.LUMINOUS.get(), 3600));
+						}
+
+						((AbstractArrowEntityAccess) abstractarrowentity).setGlow(true);
+
+						worldIn.addEntity(abstractarrowentity);
+					}
+
+					worldIn.playSound((PlayerEntity)null, playerentity.getPosX(), playerentity.getPosY(), playerentity.getPosZ(), SoundEvents.ENTITY_ARROW_SHOOT, SoundCategory.PLAYERS, 1.0F, 1.0F / (random.nextFloat() * 0.4F + 1.2F) + f * 0.5F);
```
Comment (SwanX1) [inline src/main/java/com/nekomaster1000/infernalexp/items/GlowsilkBowItem.java]:
```suggestion
					worldIn.playSound(null, playerentity.getPosX(), playerentity.getPosY(), playerentity.getPosZ(), SoundEvents.ENTITY_ARROW_SHOOT, SoundCategory.PLAYERS, 1.0F, 1.0F / (random.nextFloat() * 0.4F + 1.2F) + f * 0.5F);
```
I'm pretty sure you don't need the typecast

**Review by SwanX1 — APPROVED** [review]
LGTM

File: src/main/java/com/nekomaster1000/infernalexp/items/GlowsilkBowItem.java (hunk 1 above)
Comment (Hellion-Games) [reply src/main/java/com/nekomaster1000/infernalexp/items/GlowsilkBowItem.java]: Oh yeah, probably not. Just copied that over from BowItem, so I didn't look to closely at all of it. Should we test it before making that change or assume it's fine?

Comment (Hellion-Games) [reply src/main/java/com/nekomaster1000/infernalexp/items/GlowsilkBowItem.java]: Nevermind, IntelliJ marks it as a redundant cast. Should be fine. Committing your suggestion and then merging!

Comment (SwanX1) [reply src/main/java/com/nekomaster1000/infernalexp/items/GlowsilkBowItem.java]: I believe World.playSound doesn't have any overloads that could be ambiguous to the type of the first parameter, so casting the null is unnecessary. You can commit this without worrying

**Pull request merged**